lib/wheel-indicator.js

```diff
@@ -203,15 +203,15 @@ var WheelIndicator = (function() {
 
     function addEvent(elem, type, handler){
         if(elem.addEventListener) {
-            elem.addEventListener(type, handler, false);
+            elem.addEventListener(type, handler, { passive: false });

```

Comment:
You should use feature detection, because old browsers don't accept object as third argument. https://github.com/WICG/EventListenerOptions/blob/gh-pages/explainer.md#feature-detection